dynamic/parameterize/args.go

```diff
@@ -31,27 +31,46 @@ type RemoteArgs struct {
 	Name string
 	// Version is the (possibly empty) version constraint on the provider.
 	Version string
+	// Docs is the (possibly empty) argument to download and write docs into the schema
+	Docs bool
 }
 
 // LocalArgs represents a local TF provider referenced by path.
 type LocalArgs struct {
 	// Path is the path to the provider binary. It can be relative or absolute.
 	Path string
+	// DocsLocation is the path to the provider documentation.
+	DocsLocation string
 }
 
 func ParseArgs(args []string) (Args, error) {
 	// Check for a leading '.' or '/' to indicate a path
+
 	if len(args) >= 1 &&
 		(strings.HasPrefix(args[0], "./") || strings.HasPrefix(args[0], "/")) {
 		if len(args) > 1 {
-			return Args{}, fmt.Errorf("path based providers are only parameterized by 1 argument: <path>")
+			docsArg := args[1]
+			docsLocation, found := strings.CutPrefix(docsArg, "docsLocation=")
+			if !found {
+				return Args{}, fmt.Errorf("path based providers are only parameterized by 2 arguments: <path> [docsLocation]")
```

Comment:
If we are going to require a prefix of `docsLocation=`, it needs to be documented.
```suggestion
				return Args{}, fmt.Errorf("path based providers are only parameterized by 2 arguments: <path> [docsLocation=<localPath>]")
```